samples/app_event_manager/sample.yaml

```diff
@@ -34,7 +34,7 @@ tests:
       - nrf9160dk/nrf9160/ns
       - nrf21540dk/nrf52840
       - nrf54h20dk/nrf54h20/cpuapp
-    tags: sysbuild
+    tags: sysbuild sample_app_event_manager
```

Comment:
btw. Consider placing tags under `common` to simplify maintenance (also for test)